src/Crate/PDO/Http/ServerInterface.php

```diff
@@ -51,6 +52,21 @@ public function configure(PDOInterface $PDO): void;
      */
     public function execute(string $queryString, array $parameters): CollectionInterface;
 
+    /**
+     * Execute the PDOStatement in "bulk operations" and return the response from server
+     * wrapped inside a BulkResponseInterface.
+     *
+     * Bulk operations are only supported for `INSERT`, `UPDATE`, and `DELETE` statements.
+     *
+     * https://crate.io/docs/crate/reference/en/5.2/interfaces/http.html#bulk-operations
```

Comment:
should we maybe replace `5.2` with `latest`?

Reply:
Fixed with  3e4eb52eb59b6a.